File: src/main/java/com/dashboard/interfaces/serviceInterfaces/AttendanceServiceInterface.java
package com.dashboard.interfaces.serviceInterfaces;

import com.dashboard.models.JPA.Attendance;

import java.util.List;

public interface AttendanceServiceInterface {
    void save(Attendance attendance);
    void delete(Integer attendanceId);
    void update(Integer attendanceId, Attendance attendance);
    boolean exist(Integer attendanceId);
    List<Attendance> getAllAttendances();
    Attendance getAttendanceById(Integer attendanceId);
}

File: src/main/kotlin/dashboard/interfaces/serviceInterfaces/AttendanceServiceInterface.kt
package dashboard.interfaces.serviceInterfaces

import dashboard.models.jpa.Attendance

interface AttendanceServiceInterface {
    fun save(attendance: Attendance?)
    fun delete(attendanceId: Int?)
    fun update(attendanceId: Int?, attendance: Attendance?)
    fun exist(attendanceId: Int?): Boolean
    val allAttendances: List<Attendance?>?
    fun getAttendanceById(attendanceId: Int?): Attendance?
}